examples/07_advanced/plot_beta_series.py

```diff
@@ -126,7 +126,7 @@
 # Transform the DataFrame for LSA
 lsa_events_df = events_df.copy()
 conditions = lsa_events_df["trial_type"].unique()
-condition_counter = {c: 0 for c in conditions}
+condition_counter = dict.fromkeys(conditions, 0)
```

Comment:
I don't really like it more.

Reply:
I would say let's go for it and revisit this one and some others when we discuss if we should avoid 'complex' python code in examples to make sure we are beginner friendly.